lib/src/goldens/golden_collections.dart

```diff
@@ -12,10 +13,15 @@ import 'package:image/image.dart' as img;
 /// corresponding golden image.
 /// {@endtemplate}
 class GoldenCollection {
-  GoldenCollection(this.imagesById);
+  GoldenCollection(
+    this.imagesById, {
+    required this.metadata,
+  });
 
   final Map<String, GoldenImage> imagesById;
 
+  final GoldenSceneMetadata metadata;
```

Comment:
This shouldn't be here. I assume this was added to make the scene metadata available somewhere that we have a collection. Currently these are separate concepts. Perhaps later we'll merge them both into `GoldenSceneMetadata`, but for now we should respect the barrier.

Is there a reasonable way to get the scene metadata where you need it?